guides/advanced-use/asset-compilation.md

```diff
@@ -4,7 +4,7 @@ When working on an Ember app, sometimes you may want to customize how certain ki
 
 For information on stylesheets, please see the dedicated section, [Stylesheet compilation](../stylesheets/)
 
-## Raw Assets
+## Raw assets
```

Comment:
Was it intentional to include this change in this PR?

Reply:
no not intentional - will update